src/main/java/seedu/address/ui/StallListPanel.java

```diff
@@ -25,10 +25,31 @@ public class StallListPanel extends UiPart<Region> {
      */
     public StallListPanel(ObservableList<Stall> stallList) {
         super(FXML);
+        if (stallList.size() == 1) {
```

Comment:
Wont this always always setCellFactory with StallListViewCell since it is not if else? And even if it was if else, when theres only 1 item in the list itll always show the full stall details which is unintended